front-end-components/pipelines/front-end.yaml

```diff
@@ -0,0 +1,31 @@
+trigger:
+- main
+- feature/*
```

Comment:
Don't think we want to trigger whenever you push to your feature branch. It should be when a change is made to main only

Reply:
@michael-fielding I think this is resolved now?